packages/runtime-core/src/componentEmits.ts

```diff
@@ -76,6 +77,22 @@ export function emit(
     }
   }
 
+  let args = rawArgs
```

Comment:
maybe we should use `rawArgs` directly here

Reply:
can you explain more on how to use `rawArgs` here?